include/dlaf/communication/kernels/all_reduce.h

```diff
@@ -66,14 +65,19 @@ DLAF_MAKE_CALLABLE_OBJECT(allReduceInPlace);
 template <class CommSender, class TileInSender, class TileOutSender>
 [[nodiscard]] auto scheduleAllReduce(CommSender&& pcomm, MPI_Op reduce_op, TileInSender&& tile_in,
                                      TileOutSender&& tile_out) {
+  using dlaf::comm::CommunicationDevice_v;
   using dlaf::comm::internal::allReduce_o;
   using dlaf::comm::internal::transformMPI;
   using dlaf::internal::CopyFromDestination;
   using dlaf::internal::CopyToDestination;
   using dlaf::internal::RequireContiguous;
+  using dlaf::internal::SenderSingleValueType;
   using dlaf::internal::whenAllLift;
   using dlaf::internal::withTemporaryTile;
 
+  constexpr static auto D_IN = dlaf::internal::SenderSingleValueType<TileInSender>::D;
+  constexpr static auto D_OUT = dlaf::internal::SenderSingleValueType<TileOutSender>::D;
```

Comment:
```suggestion
  constexpr static auto DIn = dlaf::internal::SenderSingleValueType<TileInSender>::D;
  constexpr static auto DOut = dlaf::internal::SenderSingleValueType<TileOutSender>::D;
```
or
```suggestion
  constexpr static auto in_device = dlaf::internal::SenderSingleValueType<TileInSender>::D;
  constexpr static auto out_device = dlaf::internal::SenderSingleValueType<TileOutSender>::D;
```
or
https://github.com/eth-cscs/DLA-Future/blob/f56b7745c65f0c58234b1ce3b4678b90cffad9ad/include/dlaf/communication/kernels/p2p.h#L77-L78
?

Reply:
See 66ce0c6, better?

Reply:
See 9e93638, better?

Reply:
Yep, thanks!